great_expectations/data_context/data_context.py

```diff
@@ -1675,9 +1681,9 @@ def get_validator(
                 expectation_suite_name=create_expectation_suite_with_name
             )
 
-        batch: Batch = cast(
+        batch_list: Batch = cast(
```

Comment:
@talagluck This might be an error; happy to review synchronously in order to make sure that I am looking at the correct new code.  Thanks.